frontend/packages/kubevirt-plugin/integration-tests/views/virtualMachine.view.ts

```diff
@@ -73,6 +73,8 @@ export const vmDetailstatusButton = (namespace, name) =>
 
 export const vmDetailService = (serviceName) => $(`[data-test-id="${serviceName}"]`);
 
+export const vmLoggedInUsers = element(by.xpath('//a[contains(@href,"logged-in-users")]'));
```

Comment:
Can we use https://github.com/openshift/console/blob/eb8bc4314d648e3dfcd53946ca993150a203a6f1/frontend/packages/kubevirt-plugin/integration-tests/views/dashboard.view.ts#L12 here?

Reply:
why?

Reply:
One point is it consistents with others.

Reply:
I've already opened https://bugzilla.redhat.com/show_bug.cgi?id=1900377 for the missing selector, there is no point using another.